src/chargemaster_parsers/parsers/parsers.py

```diff
@@ -49,29 +49,28 @@ def parse_price(self, price):
 
 class ChargeMasterEntry:
     __slots__ = sorted(
-        [
-            "location",
-            "procedure_identifier",
-            "procedure_description",
-            "ndc_code",
-            "nubc_revenue_code",
-            "cpt_code",
-            "hcpcs_code",
-            "ms_drg_code",
-            "max_reimbursement",
-            "min_reimbursement",
-            "expected_reimbursement",
-            "in_patient",
-            "payer",
-            "plan",
-            "gross_charge",
-            "extra_data",
-            # Unused
-            "charge_code",
-            "quantity",
-            "in_patient_price",
-        ]
-    )
+      [
+        "location",
+        "procedure_identifier",
+        "procedure_description",
+        "ndc_code",
+        "nubc_revenue_code",
+        "cpt_code",
+        "hcpcs_code",
+        "ms_drg_code",
+        "max_reimbursement",
+        "min_reimbursement",
+        "expected_reimbursement",
+        "in_patient",
+        "payer",
+        "plan",
+        "gross_charge",
+        "extra_data",
+        # Unused
+        "charge_code",
+        "quantity",
+        "in_patient_price"
+    ])
```

Comment:
Did something interesting change here or just your IDE? I haven't been enforcing it but was trying to keep with the black coding style whatever that ends up being.

Reply:
i think it's just my IDE? i can exclude this file